.github/workflows/unittest.yml

```
@@ -91,13 +91,14 @@ jobs:
     - name: Find Researcher Role Source
       run: |
         echo "🔍 Hunting for the mysterious 'Researcher' role..."
+        cd src/praisonai
         python -c "
         import os
         import yaml
         import glob
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)